assets/src/edit-story/components/canvas/pagepreview/index.js

```diff
@@ -36,7 +36,7 @@ const Page = styled.button`
 	outline: ${ PAGE_THUMB_OUTLINE }px solid ${ ( { isActive, theme } ) => isActive ? theme.colors.selection : theme.colors.bg.v1 };
 	height: ${ ( { height } ) => height }px;
 	width: ${ ( { width } ) => width }px;
-	background-color: ${ ( { theme } ) => theme.colors.mg.v1 };
+	background-color: ${ ( { theme } ) => theme.colors.fg.v1 };
```

Comment:
We use foreground color for `background-color`?

Reply:
whoops!